pages/common/autossh.md

```diff
@@ -1,29 +1,29 @@
 # autossh
 
 > Runs, monitors and restarts SSH connections.
-> Auto-reconnects to keep port forwarding tunnels up. Accepts all ssh flags.
+> Auto-reconnects to keep port forwarding tunnels up. Accepts all `ssh` flags.
 > More information: <https://harding.motd.ca/autossh>.
 
-- Open an SSH session, restarting when a monitoring port fails return data:
+- Start an SSH session, restarting when a monitoring port fails return data:
 
-`autossh -M {{monitor_port}} {{ssh_command}}`
+`autossh -M {{monitor_port}} "{{ssh_command}}"`
 
-- Open an SSH session which forwards a local port to a remote one, restarting if necessary:
+- Forward a local port to a remote one, restarting when necessary:
 
 `autossh -M {{monitor_port}} -L {{local_port}}:localhost:{{remote_port}} {{user}}@{{host}}`
 
-- Fork before executing ssh (runs in the background) and don't open a remote shell:
+- Fork `autossh` into the background before executing `ssh` and don't open a remote shell:
 
-`autossh -f -M {{monitor_port}} -N {{ssh_command}}`
+`autossh -f -M {{monitor_port}} -N "{{ssh_command}}"`
 
-- Run autossh in the background, with no monitoring port, instead relying on SSH keep-alives every 10 seconds to detect failure:
+- Run in the background, with no monitoring port, instead sending SSH keep-alive packets every 10 seconds to detect failure:
 
-`autossh -f -M 0 -N -o "ServerAliveInterval 10" -o "ServerAliveCountMax 3" {{ssh_command}}`
+`autossh -f -M 0 -N -o "ServerAliveInterval 10" -o "ServerAliveCountMax 3" "{{ssh_command}}"`
 
-- Run autossh in the background, with no monitoring port, no remote shell, exiting if the port forward fails:
+- Run in the background, with no monitoring port and no remote shell, exiting if the port forward fails:
 
 `autossh -f -M 0 -N -o "ServerAliveInterval 10" -o "ServerAliveCountMax 3" -o ExitOnForwardFailure=yes -L {{local_port}}:localhost:{{remote_port}} {{user}}@{{host}}`
 
-- Run autossh in the background with debug output logged to a file and ssh verbose output logged to a second file:
+- Run in the background, logging `autossh` debug output and `ssh` verbose output to files:
 
-`AUTOSSH_DEBUG=1 AUTOSSH_LOGFILE={{log_file}} autossh -f -M {{monitor_port}} -v -E {{ssh_log_file}} {{ssh_command}}`
+`AUTOSSH_DEBUG=1 AUTOSSH_LOGFILE={{path/to/autossh_log_file.log}} autossh -f -M {{monitor_port}} -v -E {{path/to/ssh_log_file.log}} {{ssh_command}}`
```

Comment:
`VARIABLE=value` doesn't work in all shells. Is this a problem?

Reply:
It doesn't?  Do you know one in particular?

Reply:
`Fish` and maybe `dash` (didn't test it).

Reply:
Hrm, that is awlward. However unfortunately according to the man page there isn't an alternate way of specifying these parameters.

Reply:
Those are just environmental variables right? They do exist in all shells, just the syntax is different. (For example `set variable value` in fish). However I think that users of different shells should be aware of those differences.

Reply:
Yes they are env variables @marchersimon, but they're temporary, i.e. they only last for the duration of the command that command comes after them.  To demonstrate:
```console
$ env | grep ABC
$ ABC=3 env | grep ABC
ABC=3
$ env | grep ABC
$
```
Versus a normal variable (which looks to me what `set variable value` in Fish is):
```console
$ export ABC=3
$ env | grep ABC
ABC=3
$ env | grep ABC
ABC=3
$ env | grep ABC
ABC=3
$
```

Reply:
I didn't know that existed. I just tested it with Fish and if you write a command afterwards it works the same as in Bash.

Reply:
Works in `dash` and `ksh` as well.  Not `xonsh`, `tcsh`, or `csh` though.  I say we should keep it 👍🏻